tests/scripts/all.sh

```diff
@@ -222,9 +227,12 @@ cleanup()
     fi
 
     command make clean
+    cd crypto
+    command make clean
+    cd ..
```

Comment:
It would be clearer and more robust to use `command make -C crypto clean`, and likewise below `rm crypto/…` and `git -C crypto …`. This way you wouldn't need to change the shell process's state with `cd`, and risk not being able to change back or running an error handler while in an unexpected directory.

Reply:
That's a valid point but it's a pre-existing issue, so I think it should be addressed in another PR, and we should focus on just merging here. We really don't want to slow this PR down.

Reply:
Resolved here: https://github.com/ARMmbed/mbedtls/pull/2438